apps/andi/lib/andi/input_schemas/organization.ex

```diff
@@ -7,7 +7,7 @@ defmodule Andi.InputSchemas.Organization do
 
   alias Andi.InputSchemas.StructTools
 
-  @primary_key {:id, Ecto.UUID, autogenerate: false}
+  @primary_key {:id, :string, autogenerate: false}
```

Comment:
I thought we were keeping the UUID and adding validation to the changeset function